### ISSUE TITLE
Laravel 5

### ISSUE THREAD

**Issue by lehnen** [maxxscho/laravel-tcpdf#14]
maxxscho/laravel-tcpdf 0.1.0 requires illuminate/support ~4 
Will there be an update?

**Comment by maxxscho** [comment]
Yes, there will be an update. Please be patient - very very busy days. I'll have a look at this next week.
Thanks!!!

**Comment by dylanharbour** [comment]
@maxxscho , do you have any plans to extend this to L5? I'd love to use your package.

**Comment by maxxscho** [comment]
Yes I'll definitely update the package. Sadly there is an immense workload which I have to finish before. I guess it will take a few weeks or so…
So sorry to say this… :(

**Comment by rginnow** [comment]
tecnick.com/tcpdf is abandoned. use tecnickcom/tcpdf

https://packagist.org/packages/tecnick.com/tcpdf
